packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -136,6 +144,36 @@ const Tabs = ({
     }
   }, [value]);
 
+  // listen for visibility change to reset scroll buttons override
+  useEffect(() => {
+    // keep track of animation frame to cancel when needed
+    let animationFrame: number;
+
+    // called when unmounted or scroll buttons is reset
+    const cleanup = () => {
+      cancelAnimationFrame(animationFrame);
+      document.removeEventListener("visibilitychange", refreshScrollButtons);
+    };
+
+    // Reset the scroll buttons override when the document becomes visible.
+    // If called, then the document is hidden because the event listener
+    // is only registered if the document is hidden
+    function refreshScrollButtons() {
+      animationFrame = requestAnimationFrame(() => {
+        cleanup();
+        setScrollButtons("auto");
+      });
+    }
+
+    // don't override scroll buttons if it's already set to "auto"
+    if (scrollButtons !== "auto" && document.visibilityState === "hidden") {
+      document.addEventListener("visibilitychange", refreshScrollButtons);
+    }
```

Comment:
@benschell-okta had this issue as well. We need to remove this visibility check because it can happen in Monolith even when it's visible:
```suggestion
    if (scrollButtons !== "auto") {
      document.addEventListener("visibilitychange", refreshScrollButtons);
    }
```